webpack.config.js

```diff
@@ -0,0 +1,30 @@
+var path = require('path');
+
+module.exports = {
+
+    entry: path.resolve(__dirname, 'src') + '/app/index.js',
+    output: {
+        path: path.resolve(__dirname, 'dist') + '/app',
+        filename: 'bundle.js',
+        publicPath: '/app/'
```

Comment:
It can be compiled just into the `dist` folder. Don't have to add one more level